UI/RemoteISOScreen.cpp

```diff
@@ -268,19 +275,34 @@ static bool LoadGameList(const std::string &host, int port, std::vector<std::str
 	std::vector<std::string> items;
 	result.TakeAll(&listing);
 
-	SplitString(listing, '\n', items);
-	for (const std::string &item : items) {
-		if (!endsWithNoCase(item, ".cso") && !endsWithNoCase(item, ".iso") && !endsWithNoCase(item, ".pbp")) {
-			continue;
+	if (startsWith(responseHeaders[0],"Server: SuperDuperServer")) {
```

Comment:
Oh. Haha, I forgot I named the server that. Please change both this and the place where it's defined to PPSSPPServer or something.